addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/TechsJsonParser.java

```diff
@@ -93,21 +94,28 @@ TechData parse(String categories, List<String> technologies, boolean createIcons
                         .map(
                                 path ->
                                         CompletableFuture.runAsync(
-                                                () ->
-                                                        parseJson(
-                                                                techData,
-                                                                getStringResource(path),
-                                                                createIcons),
+                                                () -> {
+                                                    if (LOGGER.isDebugEnabled()) {
+                                                        String[] name = path.split("/");
+                                                        LOGGER.info(name[name.length - 1]);
+                                                    }
+                                                    parseJson(
+                                                            techData,
+                                                            getStringResource(path),
+                                                            createIcons);
+                                                },
                                                 executor))
                         .toList();
         // Note: Based on testing having the forEach separate performs faster than chaining it
         futures.forEach(CompletableFuture::join);
         executor.shutdown();
         Instant finish = Instant.now();
+        Long loadTime = Duration.between(start, finish).toMillis();
         LOGGER.info(
-                "Loaded {} Tech Detection technologies, in {}ms",
+                "Loaded {} Tech Detection technologies, in {} ({}ms)",
                 techData.getApplications().size(),
-                Duration.between(start, finish).toMillis());
```

Comment:
Why not just log the duration which follows ISO-8601? That's very easy to read.

Reply:
Do you mean `Duration.toString()` uses 8601?
Because `System.out.println(Duration.ofMillis(4005L));` prints `PT4.005S`, I mean I guess I could format it and stuff but I was trying to keep it simple.

Reply:
The leading PT threw me, I'll go read about it.